axum-extra/Cargo.toml

```diff
@@ -42,6 +42,7 @@ tower-service = "0.3"
 axum-macros = { path = "../axum-macros", version = "0.2.2", optional = true }
 cookie-lib = { package = "cookie", version = "0.16", features = ["percent-encode"], optional = true }
 percent-encoding = { version = "2.1", optional = true }
+prost = { version = "0.11.0", optional = true }
```

Comment:
Thinking we should call the feature `protobuf` instead of `prost`.

Reply:
@davidpdrsn By the way, I think we should start using namespaced features so we don't have a bunch of extra features like `axum-macros` and `cookie-libs`. Would you mind us bumping the MSRV for that?

Reply:
Sure!

Reply:
We should not rename the feature from `prost` to `protobuf`. This will confuse users.

Actually, there is a competing Protocol Buffer implementation for Rust provided by the [`grpc`](https://crates.io/crates/grpc) crate which is called [`grpc-protobuf`](https://crates.io/crates/grpc) (please notice, that the grpc crate is not an official implementation maintained by the gRPC group).

I once introduced support for both Protocol Buffer implementations in the [`opentelemetry-otlp`](https://docs.rs/crate/opentelemetry-otlp) crate. Users can choose their gRPC backend by using the feature flags`tonic` and`prost` or `grpc-sys` (for `grpc-protobuf`).

Reply:
I mean there are other serialization frameworks than serde, but the feature is still called `json`.

I say we rename it.